kernel/kernel-api/src/main/java/io/delta/kernel/client/ExpressionHandler.java

```diff
@@ -28,7 +27,9 @@
 public interface ExpressionHandler {
     /**
      * Create an {@link ExpressionEvaluator} that can evaluate the given <i>expression</i> on
-     * {@link io.delta.kernel.data.ColumnarBatch}s with the given <i>batchSchema</i>.
+     * {@link io.delta.kernel.data.ColumnarBatch}s with the given <i>batchSchema</i>. The
+     * <i>expression</i> is expected to be a scalar expression where for each one input row there
+     * is a one output value.
```

Comment:
werent we supposed to update this to explicitly specify the expression return data type?